ci/azure-win-installer-upload.yml

```
@@ -0,0 +1,33 @@
+steps:
+
+  - task: UsePythonVersion@0
+    inputs:
+      versionSpec: '2.7'
+    displayName: "Use System Python"
+
+  - task: BatchScript@1
+    inputs:
+      filename: ci/win_test_installer.cmd
+    displayName: "Test Installer"
+    condition: succeeded()
+
+  - task: GithubRelease@0
+    inputs:
+      gitHubConnection: neuronsimulator-installers
+      repositoryName: neuronsimulator/installers
+      action: edit
+      # note : if previous release doesn't exist, as we are pushing
+      # to different repository, we must need to provide commit id.
+      target: '9d7b1e26717bd207f7ae6114f78f7ab7b958c998'
+      tagSource: manual
+      tag: nightly
+      title: 'Nightly NEURON Developer Snapshot'
+      releaseNotesSource: input
+      releaseNotes: "Last NEURON Commit: $(Build.SourceVersionMessage)"
+      isPreRelease: true
+      assetUploadMode: replace
+      addChangeLog: false
+      assets: |
+        $(Build.SourcesDirectory)\nrn-nightly-AMD64.exe
+    displayName: 'Upload Installer'
+    condition: and(succeeded(), in(variables['Build.Reason'], 'Manual', 'Schedule', 'IndividualCI', 'PullRequest'))
```

```suggestion
    condition: and(succeeded(), in(variables['Build.Reason'], 'Manual', 'Schedule'))
```
reminder for myself!

---

Done in 38dbae466bb39794e73f88afb983b1b4725c2aea